upcaster/src/main/java/io/axoniq/dev/samples/upcaster/FlightDelayedEventUpcaster.java

```diff
@@ -0,0 +1,60 @@
+package io.axoniq.dev.samples.upcaster;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.axoniq.dev.samples.api.FlightDelayedEvent;
+import org.axonframework.serialization.SimpleSerializedType;
+import org.axonframework.serialization.upcasting.event.IntermediateEventRepresentation;
+import org.axonframework.serialization.upcasting.event.SingleEventUpcaster;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.invoke.MethodHandles;
+
+public class FlightDelayedEventUpcaster extends SingleEventUpcaster {
+
+    private static final String ORIGIN = "origin";
+    private static final String DESTINATION = "destination";
+    private static final String LEG = "leg";
+
+    private SimpleSerializedType sourceType =
+            new SimpleSerializedType(FlightDelayedEvent.class.getTypeName(), null);
```

Comment:
Kind of a nit, but I'd recommend using a constant as the type. If somebody moved the class from one directory to another, that would mean this upcaster wouldn't work as intended anymore.